### ISSUE TITLE
Refreshing Node.js Collection Review Process and Gathering Technical Editors

### ISSUE THREAD

**Issue by ZibbyKeaton** [nodejs/nodejs-collection#1]
TL;DR

- Recruit technical reviewers for Node.js Collection 
- Confirm updated process for review looks good to Node.js Collection group

**Background**
Node.js Collection is an existing Medium publication collection that has been a successful collaboration thus far with the Evangelism working group and Communications within the Node.js Foundation. Currently we receive around 2-5 articles per month for review: https://medium.com/the-node-js-collection/introducing-the-node-js-collection-4d0d60e72a64

However, there are not enough eyes on this from an editorial and technical review perspective. We’d like to make the review board bigger and ensure that there is always at least 2 technical editor included in the review process. We would like 3-5 consistent technical reviewers who would be committed to doing this. We are open to rotating the reviewers every quarter if the task becomes too time-consuming. 

To help us expand the review process and review board, a number of changes are being proposed to the Node.js Collection guidelines: https://medium.com/the-node-js-collection/introducing-the-node-js-collection-4d0d60e72a64

**Changes to call out include:** 

- The Node.js Foundation will continue to review posts for grammatical accuracy. The submission will then be sent to at least two technical editors who will provide comments and edits from a technical (not opinion) standpoint.

- Once the technical editors have reviewed the article, the review team will share all suggested edits with the author, who has final sign off on content.

- The review process may take between two to three weeks depending on the number of edits and revisions involved.

As extra incentive, we suggest recognizing the contributions of technical editors by: 

- Officially listing technical editors on the new Node.js Collection GitHub repo.

- Noting technical editors at the end of the Medium articles and post, if this acceptable to guest authors. 

- Naming and thanking technical editors on the Node.js Collectin Medium page 

**Next Steps:** 

- Use our networks on Twitter, other social media to recruit technical editors.

**Comment by nebrius** [comment]
cc @nodejs/community-committee @nodejs/evangelism

**Comment by gr2m** [comment]
> We would like 3-5 consistent technical reviewers who would be committed to doing this. We are open to rotating the reviewers every quarter if the task becomes too time-consuming.

What would be the minimal time commitment / week for people who are interested to help? Say someone would be okay to review max one article per week on a specific day so they can plan with it, would that work?

**Comment by ZibbyKeaton** [comment]
@gr2m given the cadence of articles at this point it would work, however if we start to get more contributions that might be a challenge. We can always cross that bridge when we get there.

**Comment by hackygolucky** [comment]
So is the next step here to PR this updated documentation to this repo so folks will sign on? I think these changes sound great.

**Comment by nebrius** [comment]
After discussion in today's CommComm meeting, we outlined some good next steps. I'm going to remove the `cc-agenda` label for now. Feel free to re-add it if you get blocked on something you need the committee to handle!

**Comment by ZibbyKeaton** [comment]
@nebrius did the next steps include outreach to find additional technical reviewers? I know that @hackygolucky and @ashleygwilliams had some thoughts on this. I also know that @vdeturckheim @thefourtheye & @fforres had expressed previously interest in helping to review. Any of you still up for it and have the bandwidth given the new review update?

**Comment by nebrius** [comment]
@ZibbyKeaton I would say the next steps are to create an onboarding path for people who want to review. I notice that the README is basically blank right now, and I think you should work on that first before starting to reach out to folks. Right now, there's no description of what the Node.js Collection is, what sort of help you need, and how to get involved, and you should have all of these things in the README and, ideally, a CONTRIBUTING guide too.

**Comment by bnb** [comment]
@nebrius I have submitted a PR (https://github.com/nodejs/nodejs-collection/pull/2) to make the README not blank 😉

Ideally we'll get that in ASAP. I'll merge it tomorrow if there are no comments to get it shipped. I'll also pull a CONTRBIUTING guide together to get it shipped and ready to go. @vdeturckheim I know you're pretty good at the editing side of this, would love to get any input you have on this as well 🙂

**Comment by dotproto** [comment]
I'm interested in helping as a editorial or technical editor. If @gr2m has time, we may even be able to collaborate as we're both in the LA area.

**Comment by ZibbyKeaton** [comment]
@svincent and @gr2m I wanted to confirm you two are still interested in helping as technical editors? If so, would you mind sending me your email addresses and I can add you to our email alias. I will also add you as editors on our Medium site and also list you as technical editors in our guidelines section.

**Comment by ZibbyKeaton** [comment]
Following up again on @svincent and @gr2m I have about six posts from folks looking to submit. Are you okay with being technical editors? And if so, can I add you to our alias or at least start sending you over content for a technical review?

**Comment by dotproto** [comment]
@ZibbyKeaton Sorry for the delay; I missed the first notification. I'm interested and am ready to help!

**Comment by ZibbyKeaton** [comment]
Awesome! What's your email address @svincent ? I'm planning to create another issue with items for review based off of my initial grammar edits. I'll put them into google document, so it is easy for you to comment/ask questions. Sound good to you?

**Comment by gr2m** [comment]
@ZibbyKeaton sorry I can’t help as technical editor, I just asked the question above for more context so I could reach out to others for help, but had to luck so far I’m afraid

**Comment by dotproto** [comment]
@ZibbyKeaton svincent@gmail.com should be good. Once I get the first email I'll set up a rule to mark others like it as important so they shuffle to the top of the stack 😄

**Comment by kinsomicrote** [comment]
I will like to help as a technical editor.
Though I got started with Nodejs recently (I don't know if that will be a challenge). I am guessing this will be a good way to start contributing to the Nodejs project, before making code contribution.

I have written a handful technical articles covering Ruby, none on Nodejs for the time being.

I will like to know the requirements needed for technical editors, and other related information.

Thanks.

**Comment by kinsomicrote** [comment]
@ZibbyKeaton is there still an opening?

**Comment by ZibbyKeaton** [comment]
@kinsomicrote yes, I think this is a great way to start contributing. Sorry for the delay in responding to you. 

In terms of requirements, we are just looking for folks that are eager to help here and can review blog within around a week or two that they receive them. Most of the articles don't need a ton of technical review, oftentimes its really expanding on or explaining a point better. 

In terms of the process for reviewing:

- People will submit their blog to medium@nodejs.org. We will review this blog and make sure that it fits within our guidelines, which can be found [here](https://medium.com/the-node-js-collection/introducing-the-node-js-collection-4d0d60e72a64). 

- From there, the Node.js Foundation reviews the posts for grammatical accuracy. 

-Then, we will send the blog post to you and Simeon to review

-Once this is done, the author reviews and accepts/rejects our suggestions; we also ask the author if they are comfortable with listing the technical reviewers name at the bottom of the post.

- In terms of editing, we are only looking for technical errors not differences on opinions.

If you would like to participate in this, I will need your email address to include on our medium alias. In addition, we have a google document where we manage our submissions (so our editorial calendar) and we also make our edits via google documents as well.

Let me know if this sounds good to you or if you have any additional questions here.

**Comment by kinsomicrote** [comment]
@ZibbyKeaton Thanks for the response, I do not have any questions for now.

Here is my email for Medium: kinsomicrote.inc[at]gmail.com
You can also use that to add me to the Google document.

Thanks.

**Comment by ZibbyKeaton** [comment]
I believe we can close this. Let me know if it needs to be reopened.
